seatunnel-connectors-v2/connector-hudi/src/main/java/org/apache/seatunnel/connectors/seatunnel/hudi/exception/HudiConnectorErrorCode.java

```diff
@@ -17,13 +17,28 @@
 
 package org.apache.seatunnel.connectors.seatunnel.hudi.exception;
 
-public class HudiPluginException extends Exception{
+import org.apache.seatunnel.common.exception.SeaTunnelErrorCode;
 
-    public HudiPluginException(String message) {
-        super(message);
+public enum HudiConnectorErrorCode implements SeaTunnelErrorCode {
+
+    CREATE_PARQUET_METADATA_FAILED("HUDI-01", "Create ParquetMetadata failed."),
+    KERBEROS_AUTHORIZED_FAILED("HUDI-02", "Kerberos Authorized failed");
```

Comment:
How about add this error code to common? I think it can be reused in many connectors